op-conductor/conductor/service.go

```diff
@@ -805,7 +872,27 @@ func (oc *OpConductor) startSequencer() error {
 		return err
 	}
 
-	oc.log.Info("starting sequencer", "server", oc.cons.ServerID(), "leader", oc.leader.Load(), "healthy", oc.healthy.Load(), "active", oc.seqActive.Load())
+	// If no rollup boost client, proceed with normal sequencer start
```

Comment:
please take a whole look at the function, I think it could be simplified and made more readable.